This was mailed to mbiebl@debian.org and is pending review.

--- tools/Makefile.am
+++ tools/Makefile.am
@@ -144,15 +144,19 @@
 	$(NULL)
 
 if ENABLE_UDEV_ACL
-udevdir = $(UDEVDIR)
 udevrulesdir = $(UDEVDIR)/rules.d
 
 dist_udevrules_DATA = 70-udev-acl.rules
-udev_PROGRAMS = udev-acl
+libexec_PROGRAMS = udev-acl
 
 udev_acl_SOURCES = udev-acl.c
 udev_acl_LDADD = $(UDEV_ACL_LIBS)
 udev_acl_CFLAGS = $(UDEV_ACL_CFLAGS)
+
+install-exec-hook:
+	mkdir -p $(DESTDIR)$(prefix)/lib/ConsoleKit/run-seat.d
+	ln -sf $(libexecdir)/udev-acl $(DESTDIR)$(prefix)/lib/ConsoleKit/run-seat.d/udev-acl.ck
+	ln -sf $(libexecdir)/udev-acl $(DESTDIR)$(UDEVDIR)/udev-acl
 endif
 
 EXTRA_DIST =				\
